src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  const chatEntries = props.entries.map((entry, index) => {
+    return (
+      <ul key={index}>
```

Comment:
Note that your code here ends up causing a slight violation of HTML expectations, as this `ul` tag has no `li` tags within it. Minor issue, though.